### PR TITLE
Fix overlapping between ScriptLink and Brand

### LINKED ISSUE #2534: Fix  ScriptLink positioning in the Brand component
**Is your feature request related to a problem? Please describe.**

While integrating `ScriptLink` to `Brand` in simorgh, The `ScriptLink` and the `Brand` overlap below 332px.

Screenshot 2019-10-31 at 13 00 56

**Describe the solution you'd like**
Fix `ScriptLink` styling so that it doesn't overlap but wraps nicely to the next line.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2534

**Overall change:**  Fix the overlap between `ScriptLink` and `Brand` on viewpoints < 332px.
**Code changes:**

- _Use flex-box to position the `Brand` and `ScriptLink`_

**Testing** 

- Regression testing on brand.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by DenisHdz** [comment]
LGTM, just one thing. When hovering on the `Brand with link`, the `Script Link` is pushed down. Can we do something to tweak this? Or we just have to accept it since the underline makes the height bigger...

hover-bug

File: packages/components/psammead-brand/README.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ The `scriptLink` can be used to render [ScriptLink](https://github.com/bbc/psamm
 | borderTop | Boolean | no | `false` | `true` |
 | borderBottom | Boolean | no | `false` | `true` |
 | scriptLink | Node | no | `null` | `<ScriptLink service='news' script={latin} href='https://www.bbc.com/serbian/lat'> Lat </ScriptLink>` |
-| dir | string | No | `'ltr' | One of `'rtl'` `'ltr' |
```
Comment (DenisHdz) [inline packages/components/psammead-brand/README.md:50]: Can you remember why were we using the `dir` prop before?

**Review by DenisHdz — APPROVED** [review]

File: packages/components/psammead-brand/README.md (hunk 1 above)
Comment (ghost) [reply packages/components/psammead-brand/README.md:50]: We were using it to position the `ScriptLink`  `left: 0` and `right:0`
https://github.com/bbc/psammead/pull/2539/files#diff-4f91a28dd0d1d26145e763893261c522L188

**Review by OlgaLyubin — APPROVED** [review]
Looks good to me

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

**Review by tochwill — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**